.goreleaser.yml

```diff
@@ -0,0 +1,21 @@
+project_name: nginx-build
+env:
+  - GO111MODULE=on
+before:
+  hooks:
+    - go mod tidy
```

Comment:
Is this mandatory? If this run on only release and dependencies are updated, is there possibility of the difference between committed files on only release?

Reply:
This command does not have to be executed. It may be better to execute it elsewhere. For example, you can use renovate.

https://github.com/catatsuy/notify_slack/blob/9a3ab4a1069ee1424a3ad726dc29e85e3030bc36/renovate.json#L6

Reply:
Then, can you remove this line once?

Reply:
@cubicdaiya I remove these lines. Can you check it?